src/django/api/models/facility_download_limit.py

```diff
@@ -0,0 +1,51 @@
+from simple_history.models import HistoricalRecords
+from django.db import models
+from django.db.models import (
+    BigAutoField,
+)
+
+
+class FacilityDownloadLimit(models.Model):
+    """
+    Stores the number of facility data downloads, the timestamp of the last download,
+    and the monthly download limit for a registered free user.
+    """
+    id = BigAutoField(
+        auto_created=True,
+        primary_key=True,
+        serialize=False
+    )
+    user = models.OneToOneField(
+        'User',
+        null=False,
+        on_delete=models.CASCADE,
```

Comment:
Does this relationship mean that if the `user` gets deleted `this` record gets deleted? Or it's the other way around?